src/stores/game.js

```diff
@@ -14,6 +14,31 @@ function queenCount(player) {
   return player.faceCards.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0);
 }
 
+class GameCard {
+  constructor(card) {
+    const str_rank =
+      {
+        1: 'A',
+        11: 'J',
+        12: 'Q',
+        13: 'K',
+      }[card.rank] ?? card.rank;
+    // Stringify Suit
+    const str_suit = ['♣️', '♦️', '♥️', '♠️'][card.suit];
+    this.createdAt = card.createdAt;
+    this.updatedAt = card.updatedAt;
+    this.id = card.id;
+    this.suit = card.suit;
+    this.rank = card.rank;
+    this.name = str_rank + str_suit;
+    this.attachments = card.attachments;
```

Comment:
Attachments are all cards so we need to map them here
```suggestion
    this.attachments = card.attachments.map(attachment => createGameCard(attachment));
```